component/admin/views/prices_detail/tmpl/default.php

```diff
@@ -56,7 +56,6 @@
                     </td>
                     <td><input class="text_area" type="number" name="product_price" id="product_price" size="10"
                                maxlength="10" min="0"
-                               oninput="validity.valid || (value='');"
```

Comment:
@NguyenBao10 if function validity.valid is wrong, please fix, remove valid checking is not best practice.

Thanks.